go.mod

```diff
@@ -8,59 +8,10 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/pkg/errors v0.9.1
+	github.com/robfig/cron/v3 v3.0.0
 	github.com/rs/zerolog v1.25.0
 	github.com/swaggo/http-swagger v1.1.1
 	github.com/swaggo/swag v1.7.3
 	github.com/threefoldtech/substrate-client v0.0.0-20211025141311-db8dc634979f
 	github.com/threefoldtech/zos v0.4.10-0.20210902091800-ae5c15350c50
 )
-
```

Comment:
When I try to do `go run cmds/....` it asks me to do `go mod tidy` to readd this part.

Reply:
updated for go 1.17